item-spec/item-spec.md

```diff
@@ -133,11 +137,11 @@ The following types are commonly used as `rel` types in the Link Object of an It
 | parent       | URL to the parent STAC [Catalog](../catalog-spec/README.md) or [Collection](../collection-spec/README.md). |
 | collection   | STRONGLY RECOMMENDED. URL to a [Collection](../collection-spec/README.md). *Absolute* URLs should be used whenever possible. The referenced Collection is STRONGLY RECOMMENDED to implement the same STAC version as the Item. |
 | derived_from | URL to a STAC Item that was used as input data in the creation of this Item. |
```

Comment:
Two questions:
1. Why is `root` not listed?
2. Should we move the whole derived_from part also to best practices? I think it would make sense.